src/functionalTest/java/uk/gov/hmcts/reform/bulkscan/orchestrator/helper/CaseDataExtractor.java

```diff
@@ -53,14 +54,14 @@ private CaseDataExtractor() {
         }
     }
 
-    private static ScannedDocument mapDocument(Document document) {
+    private static ScannedDocument mapDocument(Document document, String dmUrl) {
         return new ScannedDocument(
             document.fileName,
             document.controlNumber,
             document.type,
             document.subtype,
             ZonedDateTime.ofInstant(document.scannedAt, ZoneId.systemDefault()).toLocalDateTime(),
-            new CcdDocument(String.valueOf(document.url)),
+            new CcdDocument(StringUtils.join(dmUrl, "/", document.uuid)),
```

Comment:
we can just use java's `String` object here. Unless I'm not seeing extra bit of magic here 🤔

```suggestion
            new CcdDocument(String.join("/", dmUrl, document.uuid)),
```

Reply:
yeah`String.join` can be used here 👍